File: src/Sportoholic/wwwroot/js/models/SportItem.js
import ko from "knockout";

export default class SportItem {
	constructor(params) {
		this.id = ko.observable(params.Id);
		this.weight = ko.observable(params.Weight);
		this.walking = ko.observable(params.Walking);
		this.workout = ko.observable(params.Workout);
		this.description = ko.observable(params.Description);
		this.date = ko.observable(params.Date);
	}
}

File: src/Sportoholic/Models/SportItem.cs
using System;

namespace Sportoholic.Models
{
	public class SportItem
	{
		public int Id { get; set; }
		public int Weight { get; set; }
		public bool Walking { get; set; }
		public bool Workout { get; set; }
		public string Description { get; set; }
		public DateTime Date { get; set; }
	}
}